conf/messages.en

```diff
@@ -158,16 +158,16 @@ healthcareList1.occupation4 = nurse or dental nurse
 healthcareList1.occupation5 = occupational, speech, physio or other therapists
 healthcareList1.occupation6 = phlebotomist or radiographer
 
-typeOfTransport.title = What type of transport and distribution do you work in?
-typeOfTransport.heading = What type of transport and distribution do you work in?
-typeOfTransport.airlines = Airlines
-typeOfTransport.publicTransport = Public transport
-typeOfTransport.railways = Railways
-typeOfTransport.seamanCarpenter = Seaman carpenter
-typeOfTransport.vehicles = Vehicles
-typeOfTransport.noneOfTheAbove = None of the above
-typeOfTransport.checkYourAnswersLabel = What type of transport and distribution do you work in?
-typeOfTransport.error.required = Select what type of transport and distribution do you work in
+typeOfTransport.title = What type of transport and travel do you work in?
```

Comment:
"What type of transport or travel do you work in?" 

Looks like its a "or" instead of "and"